WEB/HTML/demo.html

```diff
@@ -3,56 +3,64 @@
 <html>
 	<head>
 		<title> Group Project Team 5 - Demo </title>
-		<link href="../CSS/style.css" rel="stylesheet" type="text/css" /> 
-		<link href="../CSS/demo_style.css" rel="stylesheet" type="text/css" /> 
-		
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<!-- styles -->
+		<link href="../CSS/style.css" rel="stylesheet" type="text/css" />
+		<link href="../CSS/demo_style.css" rel="stylesheet" type="text/css" />
+		<link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet">
+		<!-- scripts -->
+		<script src="../JS/jquery-3.3.1.min.js"></script>
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
 		<!-- JS function to show/hide the ouptputDiv element-->
-		<script src="../JS/showFunc.js">
-		</script>
-		
+		<script src="../JS/showFunc.js"></script>
 		<!-- There's another JS script at the bottom of the page-->
-		
 	</head>
 
 	<body>
-		<!-- menù at the top of the page-->
-		<nav class = "titleBar" > <ul class = "titleBarButtons"> <li>TITLE</li> <li><a href="./index.html">WHAT?</a></li> <li><a href="./demo.html">DEMO</a></li> <li><a href="./about_us.html">ABOUT US</a></li> </ul> </nav>
-		
-		<!-- div which contains the actual text of the page -->
-		<div class="page">
-		
-		<div class="dragDrop" id ="dropZone">
+		<!-- menu at the top of the page + script to load it -->
+		<div id = "demo_title_bar"> Need server for this to show up </div>
+		<script> $(function(){ $("#demo_title_bar").load("demo_title_bar.html"); }); </script> 
 
+
+		<div class="row">
+			<div class="col-sm-8">
+				<div class="dragDrop" id ="dropZone"> </div>
+			</div>
+
+			<div class="col-sm-4" style="padding-top:30px;">
```

Comment:
I'm my browser with the text of the form get cut. 
<img width="1209" alt="form" src="https://user-images.githubusercontent.com/33177749/38875449-4eff8dd2-4252-11e8-9bb2-023dd65b82e6.png">

Not sure if the same happens with the English version. If so I suggest we enlarge this <div> element. (Since this is in bootstrap we need to reduce the size of the drag&drop or move the form below it).

Reply:
I tried to see how we can do that, and we can simply increate the width of the element in the demo_styles.css